galpy/potential/Potential.py

```diff
@@ -2133,6 +2128,12 @@ def _evaluateRforces(Pot, R, z, phi=None, t=0.0, v=None):
         raise PotentialError(
             "The (list of) Potential instances includes dissipative components, but you did not provide the 3D velocity (required for dissipative forces)"
         )
+    return _evaluateRforces(Pot, R, z, phi=phi, t=t, v=v)
+
+
+def _evaluateRforces(Pot, R, z, phi=None, t=0.0, v=None):
+    """Raw, undecorated function for internal use"""
+    isList = isinstance(Pot, list)
```

Comment:
It looks like this function (and the others similar) could be simplified to

```
    isList = isinstance(Pot, list)
    if isList:
        out = 0.0
        for pot in Pot:
            if pot.isDissipative:
                out += pot._Rforce_nodecorator(R, z, phi=phi, t=t, v=v)
            else:
                out += pot._Rforce_nodecorator(R, z, phi=phi, t=t)
        return out
    elif Pot.isDissipative:
        return Pot._Rforce_nodecorator(R, z, phi=phi, t=t, v=v)
    else:
        return Pot._Rforce_nodecorator(R, z, phi=phi, t=t)
```

Does this retain the same logic? I.e., is `isinstance(Pot, Potential)` essentially just checking that it is non-Dissipative?

Perhaps it would be better if I left the `else: PotentialError` for now, just in case?

Thought I'd best check before I went ahead and made further changes, because there are several similar instances of this code block.